cmd/gitops/create/tenants/cmd.go

```diff
@@ -0,0 +1,56 @@
+package tenants
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/weaveworks/weave-gitops/pkg/kube"
+	"github.com/weaveworks/weave-gitops/pkg/tenancy"
+)
+
+type tenantCommandFlags struct {
+	filename string
+	export   bool
+}
+
+var flags tenantCommandFlags
+
+var TenantsCommand = &cobra.Command{
+	Use:   "tenants",
+	Short: "create and update tenant resources",
+	RunE:  createTenantsCmdRunE(),
+}
+
+func init() {
+	TenantsCommand.PersistentFlags().StringVar(&flags.filename, "filename", "", "the file containing the tenant declarations")
```

Comment:
I wonder if we should make this `--from-file=<file-path>`, but also support creating a tenant without a tenant file - e.g. `gitops create tenant --name <name> --namespace <namespace>`

Reply:
Nice 🙂 I will add this too.

Reply:
I think accepting an array of namespaces would be good!

Reply:
Makes sense to me :)